addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -250,6 +255,16 @@ export class WebsocketWorker {
         return this.websocket && this.websocket.readyState === 0;
     }
 
+    /**
+     * Determine whether or not the websocket associated to this worker
+     * is closed.
+     *
+     * @returns {boolean}
+     */
+    _isWebsocketClosed() {
+        return this.websocket && this.websocket.readyState === 3;
```

Comment:
magic number, I know what readystate 4 is but not 3, had t google it
<https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/readyState>

> 3 | LOADING | Downloading; responseText holds partial data.

how does LOADING means closed??

Reply:
Wrong API :)  https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/readyState

Reply:
a least add a comment "// 3 is ClOSED"

Reply:
Don't you think `_isWebsocketClosed` and the docstring are enough?